drivers/clock_control/clock_control_mcux_ccm_rev2.c

```diff
@@ -225,11 +225,8 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 #endif
 
 #ifdef CONFIG_MCUX_FLEXIO
-	case IMX_CCM_FLEXIO1_CLK:
-		clock_root = kCLOCK_Root_Flexio1;
-		break;
-	case IMX_CCM_FLEXIO2_CLK:
-		clock_root = kCLOCK_Root_Flexio2;
+	case IMX_CCM_FLEXIO_CLK:
```

Comment:
I don't see this defined in `include/zephyr/dt-bindings/clock/imx_ccm_rev2.h`. You also need to update the dts file where this is used i.e: `./dts/arm/nxp/nxp_rt11xx.dtsi`

Reply:
I don't see this defined in include/zephyr/dt-bindings/clock/imx_ccm_rev2.h
what do you mean?

I found below in imx_ccm_rev2.h
/* FLEXIO */
#define IMX_CCM_FLEXIO_CLK             0x1700UL
#define IMX_CCM_FLEXIO1_CLK            0x1700UL
#define IMX_CCM_FLEXIO2_CLK            0x1701UL